aiodocker/events.py

```diff
@@ -40,10 +45,15 @@ def _transform_event(self, data):
         }
         params = ChainMap(forced_params, params)
         try:
+            # timeout has to be set to 0, None is not passed
+            # Otherwise after 5 minutes the client
+            # will close the connection
+            # http://aiohttp.readthedocs.io/en/stable/client_reference.html#aiohttp.ClientSession.request
             response = await self.docker._query(
                 "events",
                 method="GET",
                 params=params,
+                timeout=0
```

Comment:
I just checked the code of aiohttp and got very confused with the implementation of the timeout. I suppose this code will work but I have to admit the use of a sentinel would have been more clear to me because more consistent with the implementation of the method `request` on `ClientSession`. The implementation of `TimeoutHandle` is an implementation detail to me. On the other hand I checked the documentation of aiohttp and it's even more confusing to me...

@asvetlov can I adapt the documentation of the timeouts on aiohttp? It's using a sentinel everywhere but when you look at the doc it seems to be using `None`. This is just not accurate to me.

Also I'm not a big fan of the behavior implemented twice for the timeout: when the value is 0 and when the value is `None` but I suppose it's too late for that at this point. I would say pick one and drop the other. Maybe we can deprecate one of them already?

https://github.com/aio-libs/aiohttp/blob/master/aiohttp/helpers.py#L677

Reply:
(Also I would add a comma at the end of the line since the closing parenthesis is on the next line)

Reply:
So you would prefer to use timeout=None?

Reply:
Yes and no... it depends on what Andrew will say of the future of this timeout parameter. If it stays as it is now I think both are fine. But this PR is a bug fix imho so we shouldn't wait to merge, we can change it later if necessary.